changes/ce/fix-10117.en.md

```diff
@@ -0,0 +1,2 @@
+Fix an error occurring when a joining node doesn't have plugins that are installed on other nodes in the cluster.
```

Comment:
Tip: deepl.com usually does a pretty ok job translating changelogs to Chinese.

Reply:
Thanks, I followed the latest instruction from @id to add only English changes and than have a batch translation before a release.